### PR TITLE
JWNG-536 - support of jaaz

### PR DESCRIPTION
**Pull request by mignonat**

### CODE REVIEW

File: public/core.d.ts [hunk 1]
```diff
@@ -9183,6 +9207,140 @@ declare namespace JMap {
...
+       * console.log(`myObjects: [${myObjects.join(", "")}]`)
+       * // display message 'myObjects: [{ \"id\": 1, \"name\": \"Green\" }, { \"id\": 3, \"name\": \"Blue\" }]'
+       * ```
+       */
+      function removeByProperty<T extends object>(array: T[], propertyName: string, value: any, nonStrict?: boolean): T[]
+
+      /**
+       * **JMap.Util.Array.getCopyWithoutDuplicate**
+       * 
+       * Remove the first occurence.
```
Comment (TomAvediss) [inline public/core.d.ts:9328]: Get a copy of the given array without duplicate.

File: public/core.d.ts [hunk 2]
```diff
@@ -9183,6 +9207,140 @@ declare namespace JMap {
...
+     * **JMap.Util.Array**
+     * 
+     * Here you'll find all array related methods
+     */
+    namespace Array {
+
+      /**
+       * **JMap.Util.Array.remove**
+       * 
+       * Remove given elements in given array.
```
Comment (jean-kiepura) [inline public/core.d.ts:9220]: Remove given element**~~s~~** in

pas de "s" à elements

File: public/core.d.ts [hunk 3]
```diff
@@ -9183,6 +9207,140 @@ declare namespace JMap {
...
+       * console.log(`Now my numbers are [${myNumbers.join(", ")}]`)
+       * // display message "Now my numbers are [3, 5, 7]"
+       * ```
+       */
+      function remove<T>(array: T[], element: T): T[]
+
+      /**
+       * **JMap.Util.Array.findByProperty**
+       * 
+       * Remove given elements in given array.
```
Comment (jean-kiepura) [inline public/core.d.ts:9239]: Mauvaise description

File: public/core.d.ts [hunk 4]
```diff
@@ -9183,6 +9207,140 @@ declare namespace JMap {
...
+       * console.log(`Found object: "${JSON.parse(foundObject)}"`)
+       * // display message 'Found object: "{ \"id\": 2, \"name\": \"Red\" }"'
+       * ```
+       */
+      function findByProperty<T extends object>(array: T[], propertyName: string, value: any): T | undefined
+
+      /**
+       * **JMap.Util.Array.findIndexByProperty**
+       * 
+       * Remove given elements in given array.
```
Comment (jean-kiepura) [inline public/core.d.ts:9268]: Mauvaise description

**Pull request merged**